.github/workflows/main.yml

```diff
@@ -16,5 +16,4 @@ jobs:
       - uses: actions/checkout@v3
       - uses: ./.github/actions/ci-setup
 
-      - run: pnpm build
-      - run: pnpm checks
+      - run: pnpm turbo run size-test build checks
```

Comment:
If you combine these into a single `turbo` run then we build and process the graph exactly once.